peps/pep-0694.rst

```diff
@@ -435,6 +432,19 @@ the following keys:
     wishes to communicate to the end user.  These notices are specific to the overall session, not
     to any particular file in the session.
 
+.. _publishing-session-multiple:
+
+Multiple Session Creation Requests
+++++++++++++++++++++++++++++++++++
+
+If a second session is created for the same name-version pair while a session for that pair is in the
```

Comment:
Not sure how to reword this exactly, but this may suggest that it actually gets created. I think you wanted to say that an attempt to create a second session is made.